apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.v3.$projectParam.concurrency/route.tsx

```diff
@@ -80,12 +80,10 @@ export default function Page() {
         <div className="flex flex-col">
           <Table containerClassName="border-t-0">
             <TableHeader>
-              <TableRow>
-                <TableHeaderCell>Environment</TableHeaderCell>
-                <TableHeaderCell alignment="right">Queued</TableHeaderCell>
-                <TableHeaderCell alignment="right">Running</TableHeaderCell>
-                <TableHeaderCell alignment="right">Concurrency limit</TableHeaderCell>
-              </TableRow>
+              <TableHeaderCell>Environment</TableHeaderCell>
+              <TableHeaderCell alignment="right">Queued</TableHeaderCell>
+              <TableHeaderCell alignment="right">Running</TableHeaderCell>
+              <TableHeaderCell alignment="right">Concurrency limit</TableHeaderCell>
```

Comment:
"You must place `<tr>` elements inside `<thead>` to contain your `<th>` cells. The correct structure is: `<thead><tr><th>Header Cell</th></tr></thead>`. [1](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead) [2](https://www.w3.org/TR/html401/struct/tables.html)"